provider/cluster_resource.go

```diff
@@ -715,6 +725,13 @@ func (r *ClusterResource) populateState(object *cmv1.Cluster, state *ClusterStat
 			Value: sts.InstanceIAMRoles().WorkerRoleARN(),
 		}
 
+		thumbprint, err := getThumbprint(sts.OIDCEndpointURL())
+		if err == nil {
```

Comment:
What happens if there is an error while populating the state? Is it not a problem not having the thumbprint populated?

Just wanted to make sure, atm this didn't call anything that could have an error. But usually when there's an error there's a response everywhere else

Reply:
In that case `state.Sts.Thumbprint` would be empty - I can write it explicit